fudge-ratio/src/main/webapp/calculator/main.js

```diff
@@ -1,80 +1,27 @@
+import { saveSessionState, recoverSessionState } from "../code/save.js";
```

Comment:
I wasn't familiar with export and import before this so I just looked it up. Thanks for introducing something new

Reply:
Thanks 😊, modules have a large history in Javascript, the thing is that they had just been standardized in the last ~3 years (ECMAscript modules)

Here is a great link to learn more:
https://eloquentjavascript.net/10_modules.html#h_hF2FmOVxw7